bears/general/LineBreakBear.py

```diff
@@ -0,0 +1,133 @@
+from queue import Queue
+
+from coalib.results.AbsolutePosition import AbsolutePosition
+from coalib.results.Diff import Diff
+from coalib.bearlib.languages.LanguageDefinition import LanguageDefinition
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result, RESULT_SEVERITY
+from coalib.results.SourceRange import SourceRange
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
+
+from bears.general.AnnotationBear import AnnotationBear
+from bears.general.RangeHelpers import get_specified_block_range
```

Comment:
hmm you have a point, or maybe we can move that out later? about the ordering of commits locally i have this commit after the other one. IDK whats wrong with guthub UI